git-extra/git-extra.install.in

```diff
@@ -230,6 +230,18 @@ test -d "$TMPDIR" || test ! -d "$TMP" || {\
 	#
 	# As we no longer need it, delete it.
 	test ! -h /ssl || rm /ssl
+
+	# Allow an i686 version of Git for Windows, augmented by `/arm64/*`, to serve
+	# as sort of an ARM64 version of Git for Windows (because Windows/ARM64 can run
+	# i686 executables via a built-in emulator).
+	test i686 != "$arch" ||
```

Comment:
Not sure about this line. Using the 64-bit SDK on another machine and then it doesn't work. When I remove this line, `/etc/profile` is updated correctly in the SDK root folder 🚀 Benefit of removing this line is that this patch will be applied both to `git-sdk-32` and `git-sdk-64`, so then it doesn't matter which SDK folks use to build Git. This shouldn't hurt anyone unless their MSYSTEM is set to `ARM64` 😊 

Will do some more tests later today, but at least for now `/etc/profile` is updated correctly 🎉 

![image](https://user-images.githubusercontent.com/17739158/105512959-0ca59d80-5cd2-11eb-8994-b3aaaffd0db2.png)

Reply:
> Not sure about this line. Using the 64-bit SDK on another machine and then it doesn't work.

That's on purpose, actually: you will [not find the `case` statement in the 64-bit SDK](https://github.com/git-for-windows/git-sdk-64/blob/HEAD/etc/profile). Instead, it was moved to [`/etc/msystem`](https://github.com/git-for-windows/git-sdk-64/blob/HEAD/etc/msystem) there.

The reason why the 32-bit SDK has a different `/etc/profile` is that its _MSYS_ part hasn't been updated in a _long_ time. In other words, it does not reflect the newest `filesystems` package definition.

> When I remove this line, `/etc/profile` is updated correctly in the SDK root folder 🚀 Benefit of removing this line is that this patch will be applied both to `git-sdk-32` and `git-sdk-64`, so then it doesn't matter which SDK folks use to build Git.

For the moment, I would rather leave out the complication of dealing with both `git-sdk-32` and `git-sdk-64`, but focus on getting it working correctly only in the `git-sdk-32` case.

Reply:
> for now `/etc/profile` is updated correctly

🎉

My one concern is that the `MANPATH` is now set exclusively to `/arm64/{local,share}/man`, but there is nothing there...

Reply:
So if I understand it correctly, `MANPATH` is for the Git for Windows manual files (https://git-scm.com/docs/git#Documentation/git.txt---man-path)? When is this variable actually being used?

Reply:
I created a portable installer and it correctly includes the updated `/etc/profile` 🚀 

What would be the correct order of steps to build a portable installer with the arm64 binaries?
- `sdk build mingw-w64-git`
- `pacman -U mingw-w64-i686-git-2.30.0.2.f8cbc844b8-1-any.pkg.tar.xz`
- `sdk build git-extra` (so that `/etc/profile` gets overwritten to include ARM64)
- (create the arm64 binaries here and output to `C:\git-sdk-32\arm64`)
- Build the portable installer which will also copy files from `arm64`

Reply:
It WORKS when I create a portable installer with the steps I described above! Very exciting! 🎉 

![image](https://user-images.githubusercontent.com/17739158/105546907-37a5e680-5cfe-11eb-9fa9-a1486ccede83.png)

Reply:
> 
> 
> So if I understand it correctly, `MANPATH` is for the Git for Windows manual files (https://git-scm.com/docs/git#Documentation/git.txt---man-path)? When is this variable actually being used?

When you call something like `man bash` in the SDK. This issue is not _that_ important given that we ship Git for Windows without `man.exe`, but still...

Reply:
Shall we leave it as-is for now then?

Just to confirm, this is the updated section in `/etc/profile` after running `sdk build git-extra`:

```
MINGW32)
  MINGW_MOUNT_POINT="${MINGW_PREFIX}"
  PATH="${MINGW_MOUNT_POINT}/bin:${MSYS2_PATH}${ORIGINAL_PATH:+:${ORIGINAL_PATH}}"
  PKG_CONFIG_PATH="${MINGW_MOUNT_POINT}/lib/pkgconfig:${MINGW_MOUNT_POINT}/share/pkgconfig"
  ACLOCAL_PATH="${MINGW_MOUNT_POINT}/share/aclocal:/usr/share/aclocal"
  MANPATH="${MINGW_MOUNT_POINT}/local/man:${MINGW_MOUNT_POINT}/share/man:${MANPATH}"
  ;;
MINGW64)
  MINGW_MOUNT_POINT="${MINGW_PREFIX}"
  PATH="${MINGW_MOUNT_POINT}/bin:${MSYS2_PATH}${ORIGINAL_PATH:+:${ORIGINAL_PATH}}"
  PKG_CONFIG_PATH="${MINGW_MOUNT_POINT}/lib/pkgconfig:${MINGW_MOUNT_POINT}/share/pkgconfig"
  ACLOCAL_PATH="${MINGW_MOUNT_POINT}/share/aclocal:/usr/share/aclocal"
  MANPATH="${MINGW_MOUNT_POINT}/local/man:${MINGW_MOUNT_POINT}/share/man:${MANPATH}"
  ;;
ARM64)
  MINGW_MOUNT_POINT="/arm64"
  PATH="${MINGW_MOUNT_POINT}/bin:/mingw32/bin:${MSYS2_PATH}${ORIGINAL_PATH:+:${ORIGINAL_PATH}}"
  PKG_CONFIG_PATH="${MINGW_MOUNT_POINT}/lib/pkgconfig:${MINGW_MOUNT_POINT}/share/pkgconfig"
  ACLOCAL_PATH="${MINGW_MOUNT_POINT}/share/aclocal:/usr/share/aclocal"
  MANPATH="${MINGW_MOUNT_POINT}/local/man:${MINGW_MOUNT_POINT}/share/man:${MANPATH}"
  ;;
```

Reply:
I wasn't happy leaving it as-is, in particular the rather non-intuitive `sed` invocation. @dennisameling could you please test again, and also sanity-check the new revision (it should be eminently more readable than before)?

Reply:
@dscho just tested again, the results:

✔️ Running `sdk build git-extra` and then `pacman -U git-extra-...-tar.xz` correctly updates `/etc/profile` in `git-sdk-32`:

```
ARM64)
  MINGW_MOUNT_POINT="/arm64"
  PATH="${MINGW_MOUNT_POINT}/bin:/mingw32/bin:${MSYS2_PATH}${ORIGINAL_PATH:+:${ORIGINAL_PATH}}"
  PKG_CONFIG_PATH="${MINGW_MOUNT_POINT}/lib/pkgconfig:${MINGW_MOUNT_POINT}/share/pkgconfig:/mingw32/lib/pkgconfig:/mingw32/share/pkgconfig"
  ACLOCAL_PATH="${MINGW_MOUNT_POINT}/share/aclocal:/mingw32/share/aclocal:/usr/share/aclocal"
  MANPATH="${MINGW_MOUNT_POINT}/local/man:${MINGW_MOUNT_POINT}/share/man:/mingw32/local/man:/mingw32/share/man:${MANPATH}"
  ;;
```

✔️ Things looking good in ARM64 Git:

![image](https://user-images.githubusercontent.com/17739158/105905706-10635800-6023-11eb-98de-d813c37573dd.png)

Thanks! And yes, the code is much easier to understand now, thank you 😅

Reply:
Perfect. Time to merge, then.